### PR TITLE
Added logging for `DatasetNotFound` error

### PR DESCRIPTION
**Pull request by klakhov**
### Motivation and context

This PR adds logs for further invistigation of `DatasetNotFound` error which frequently appears in case of importing datasets in various formats

### Checklist

- [x] I submit my changes into the `develop` branch
- ~~[ ] I have created a changelog fragment ~~
- ~~[ ] I have updated the documentation accordingly~~
- ~~[ ] I have added tests to cover my changes~~
- ~~[ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- ~~[ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/cvat-ai/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/cvat-ai/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/cvat-ai/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/cvat-ai/cvat/tree/develop/cvat-ui#versioning))~~

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/cvat-ai/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

## Summary by CodeRabbit

- **New Features**
  - Added detailed error logging for dataset import errors to enhance troubleshooting and user feedback.

- **Enhancements**
  - Introduced a new logging configuration for handling dataset import errors, including file location and formatting details.

- **Configuration Changes**
  - Added an environment variable `CVAT_LOG_IMPORT_ERRORS` to control the logging of dataset import errors, set to `'true'` by default in the Docker configuration.

### CODE REVIEW

File: cvat/settings/base.py [hunk 1]
```diff
@@ -503,6 +515,8 @@ class CVAT_QUEUES(Enum):
     },
 }
 
+LOG_IMPORT_ERRORS = True
```
Comment (azhavoro) [inline cvat/settings/base.py:518]: I would suggest starting all cvat related options with the CVAT_ prefix. Also, please add support for configuring this via an environment variable, it's much easier in some cases

File: cvat/requirements/base.in [hunk 2]
```diff
@@ -7,6 +7,7 @@ boto3==1.17.61
 clickhouse-connect==0.6.8
 coreapi==2.3.3
 datumaro @ git+https://github.com/cvat-ai/datumaro.git@82982b16b178eb3f39c707795bb68a3306610abf
+directory-tree==0.0.4
```
Comment (azhavoro) [inline cvat/requirements/base.in:10]: Unfortunately, this is a GPL-licensed package, so we can't use it

File: cvat/apps/engine/utils.py [hunk 3]
```diff
@@ -395,3 +396,23 @@ def build_annotations_file_name(
...
+    if current_depth > 0:
+        output += "|  " * (current_depth - 1) + "|- "
+
+    output += os.path.basename(path) + "/\n"
+
+    if max_depth is None or current_depth < max_depth:
+        for item in os.listdir(path):
+            full_path = os.path.join(path, item)
+            if os.path.isdir(full_path):
+                output += directory_tree(full_path, max_depth, current_depth + 1)
```
Comment (azhavoro) [inline cvat/apps/engine/utils.py:415]: Wouldn't using os.walk be simpler and more efficient way here?

**Comment by nmanovic** [comment]
@coderabbitai help

**Comment by nmanovic** [comment]
@coderabbitai review

File: cvat/apps/engine/utils.py (hunk 3 above)
Comment (klakhov) [reply cvat/apps/engine/utils.py]: Alright, Ive tried to use walk from datumaro which is based on os.walk and also supports depth

**Review by azhavoro — APPROVED** [review]

**Pull request merged**